grain/_src/python/dataset/stats_test.py

```diff
@@ -24,6 +25,7 @@
 from absl import flags
 from absl.testing import flagsaver
 import cloudpickle
+from grain.conftest import RUN_IN_PYTEST
```

Comment:
could you please update these imports to import the module instead, i.e.

from grain import conftest

our style guide discourages importing members